gui-apps/waybar/waybar-0.9.5-r1.ebuild

```diff
@@ -50,6 +50,8 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
+PATCHES=( "${FILESDIR}/${P}-fmt-8.0.0.patch" )
```

Comment:
Just a nitpick, not necessary but it's nice to have the patch on its own line, fits better with what other ebuilds do and easier to add more or remove without restyling, e.g.
```
PATCHES=(
    file.patch
)
```